lm_eval/tasks/wmdp/_wmdp.yaml

```diff
@@ -0,0 +1,5 @@
+group: wmdp
```

Comment:
You could also remove this file and add `group: wmdp` to the `_default_template_yaml`

Reply:
changed this is newest commit

Reply:
oh wait there is an error I think